test/index.spec.js

```diff
@@ -0,0 +1,52 @@
+const path = require('path');
+const execa = require('execa');
```

Comment:
Never heard of this dep, but it sounds interesting! `child_process` is a pain to work with.